### PR TITLE
Kovri Site Redesign

### PR DESCRIPTION
**Pull request by rehrar**
-Added HTML/CSS framework
-Restructured top level navigation
-Added multi-language functionality
-Small optimizations and fixes within multi-language

### CODE REVIEW

File: Kovri Website Instructions.md [hunk 1]
```diff
@@ -236,15 +236,15 @@ Here, the various layouts are listed that you can take advantage of, as well as
...
 Currently, the following layouts are available to choose from:
   * default: this layout is a single full-width info block as seen in [the Kovri homepage](https://getkovri.org). Nothing fancy. Should work for the majority of text-based pages.
   
 _More layouts to be added later_
 
-## Questions & Contact
+## 4. Questions & Contact
 That should be everything that you need. There will be an assets document released as well for the framework at large that will show how to make your own layouts by simply copy and pasting out of the document. This instructions document will be updated with the link when it is released.
 
 Once again, I am rehrar. Feel free to contact me with any questions regarding translation, page-creation, content, markdown, jekyll, html/css, or anything else.
```
Comment (anonimal) [inline Kovri Website Instructions.md]: We should simply direct to `#kovri-dev` and/or all the contact info in the kovri repo.

File: Kovri Website Instructions.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+Feel free to skip down to a relevant section if you already know what you need. Also, any help translating this document into another language is greatly appreciated.
+
+If for any reason you have questions or need to contact me, you can find me on the Monero Slack, reddit, and sometimes the various IRC channels using the same name: rehrar. There's a second 'r' after the 'h'. Don't forget. ;)
+
+If you're also feeling generous, a donation for my work on the website and documentation is apprecaited. Although don't forget to donate to getmonero.org also. Both addresses are provided below:
+
+rehrar XMR address: 44LGJKw6Eda72gGDaiVkB9aVq7aYpCHuC1sFFixT8FNB5vLQ1LTXztaMtYvib3QetCddxFGXZvoWDGEBvcghPLSbTaBbFpY
+
+getmonero.org XMR address: 44AFFq5kSiGBoZ4NMDwYtN18obc8AemS33DBLWs3H7otXft3XjrpDtQGv7SqSsaBYBb98uNbr2VBBEt7f2wfn3RVGQBEP3A
```
Comment (anonimal) [inline Kovri Website Instructions.md]: Re: line 34-38, as we discussed on IRC:

- This is what the FFS is for
- The fact that other people will be contributing to the site (so why would you be paid for their work?)

File: Kovri Website Instructions.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+1. Questions & Contact
+
+## 1. Intro
+Hey everyone, I'm rehrar. The kid responsible for making the custom framework and design that is used in both the getmonero.org and getkovri.org websites. If you're reading this you probably want to know how to edit a page or add content, maybe even take content away. Let me be the first to say, thank you so much! The Monero Project is made and run by the community, and you're helping to make it better. 
+
+There are a couple of differences between the Kovri and Monero websites, so I just want to emphasize *you are reading the Kovri website instructions*.
+
+Feel free to skip down to a relevant section if you already know what you need. Also, any help translating this document into another language is greatly appreciated.
+
+If for any reason you have questions or need to contact me, you can find me on the Monero Slack, reddit, and sometimes the various IRC channels using the same name: rehrar. There's a second 'r' after the 'h'. Don't forget. ;)
```
Comment (anonimal) [inline Kovri Website Instructions.md]: I think you should use the generic contact info as prescribed in the kovri repo (i.e., `#kovri-dev`).

File: Kovri Website Instructions.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+### 3.i Layouts
+Currently, the following layouts are available to choose from:
+  * default: this layout is a single full-width info block as seen in [the Kovri homepage](https://getkovri.org). Nothing fancy. Should work for the majority of text-based pages.
+  
+_More layouts to be added later_
+
+## 4. Questions & Contact
+That should be everything that you need. There will be an assets document released as well for the framework at large that will show how to make your own layouts by simply copy and pasting out of the document. This instructions document will be updated with the link when it is released.
+
+Once again, I am rehrar. Feel free to contact me with any questions regarding translation, page-creation, content, markdown, jekyll, html/css, or anything else.
```
Comment (anonimal) [inline Kovri Website Instructions.md]: I think you should keep POC (point of contact) generic (like `#kovri-dev`).

File: _site/USER_GUIDE.html [hunk 5]
```diff
@@ -162,14 +364,38 @@ <h3 id="custom-settings">Custom Settings</h3>
...
+        <div class="row center-xs">
+            <p>
+                <a class="white" href="https://github.com/anonimal/kovri-site" target="_blank">Source code</a> | 
+                <a class="white" href="https://github.com/anonimal/kovri-site/blob/master/legal/terms.md" target="_blank">Terms of use</a> | 
+                <a class="white" href="https://github.com/anonimal/kovri-site/blob/master/legal/privacy.md" target="_blank">Privacy policy</a> |
+                <a href="acknowledgements.html" class="white" target="_blank">Acknowledgements</a>
+            </p>
+        </div> 
+         <div class="row center-xs">
+               <p><a href="https://github.com/anonimal/kovri-site/blob/master/legal/copyright.md" target="_blank">Copyright</a>
```
Comment (anonimal) [inline _site/USER_GUIDE.html]: General comment: all instances of `anonimal/kovri-site` should be renamed to `monero-project/kovri-site` once that repo is ready and online.

**Comment by anonimal** [comment]
@rehrar another big question: as discussed with @alvinjoelsantos, if we want to use the markdown that's in kovri repo's doc directory, I think there are better ways to keep the site updated with those docs then by manually updating/uploading static documents. One possible solution: pulling them from a kovri docs submodule with an automated backend script. Let's talk more on IRC.

Referencing https://github.com/monero-project/kovri/issues/46

**Comment by rehrar** [comment]
So I did it, but I did it in two commits, because, once again, I'm an idiot. But I'm getting the hang of this now. :D Thanks for your patience.

**Pull request merged**